### ISSUE TITLE
[SUGGESTION] Add support for raw text components

### ISSUE THREAD

**Issue by envizar** [ShaneBeee/SkBee#523]
### Suggestion

Add `raw text component`
Because when you try to make `text component from raw "<##FF0000>Test"` and send it, it sends as red "Test"
But I need "<##FF0000>Test" text component, not red "Test" text component

### Alternatives

�

**Comment by envizar** [comment]
it still doesn't support "&" color codes

**Comment by envizar** [comment]
@ShaneBeee Issue isn't closed
![изображение](https://github.com/ShaneBeee/SkBee/assets/87126318/8e23c463-a356-4f08-b636-e4d07555889a)
![изображение](https://github.com/ShaneBeee/SkBee/assets/87126318/3bbba369-0801-445f-bc2d-a8582faed2bc)

**Comment by ShaneBeee** [comment]
you don't need to tag me, I already get notifications for all messages in these threads.
That said, your original issue was closed because what you were asking for I ended up doing a workaround a different way.
As for this issue, internally I have & and § replace (due to how components work), which I can't remove or it'll break something else.

That said, I can't create the expression `raw text component...` as Skript just completely ignores that and tries to make it a raw string. 
If you have another suggestion for an expression I can make, Im all ears.
